M2/Macaulay2/tests/normal/mutmat.m2

```diff
@@ -59,3 +59,7 @@ Q = mutableMatrix id_(R^2)_P
 assert(L_(0, 1) == 0)
 assert(U_(1, 0) == 0)
 assert(Q * L * U == M)
+
+-- issue #3709 (these used to crash M2)
+try mutableMatrix(ZZ, -1, 1)
+try mutableIdentity(ZZ, -1)
```

Comment:
These aren't sufficient tests, because they could return something without crashing or throwing an error and the test would pass. Try this instead:
```suggestion
assert try ( mutableMatrix(ZZ, -1, -1); false ) else true
assert try ( mutableMatrix(ZZ, -1,  1); false ) else true
assert try ( mutableMatrix(ZZ,  1, -1); false ) else true
assert try ( mutableIdentity(ZZ, -1);   false ) else true
```

Reply:
That commit suggestion feature is so cool!  You're even a co-author on the commit -- neat!

Reply:
Huh I didn't know that!